runtime/runtime.go

```diff
@@ -137,14 +138,15 @@ func (a *Applet) Run(config map[string]string, initializers ...ThreadInitializer
 			var starlarkVal starlark.Value
 			starlarkVal = starlark.String(v)
 
-			if a.schema != nil {
+			if strings.HasPrefix(k, "$") {
```

Comment:
I could also see an argument for dropping support for $tz and always requiring a location to be added??

Reply:
I think that would make a lot of sense. Larger change that we should discuss and tackle separately though.